patientsearch/models/sync.py

```diff
@@ -90,6 +92,16 @@ def HAPI_request(
             level="error",
         )
         raise ValueError(err)
+
+    # Fencing out - too much noise.  All API endpoints should audit when appropriate
+    if False and VERB != "GET":
```

Comment:
Maybe add a TODO so this is easier to find?

Reply:
well, it's not unfinished work, i just left behind fenced out for debugging purposes.  i'd like to keep the TODO list short and meaningful